src/WebView.tsx

```diff
@@ -1,5 +1,14 @@
-// This files provides compatibility without tree platform.
-import { WebView } from 'react-native';
+import React from 'react';
+import { Text, View } from 'react-native';
+import { IOSWebViewProps, AndroidWebViewProps } from './WebViewTypes';
+
+export type WebViewProps = IOSWebViewProps & AndroidWebViewProps;
+
+const WebView: React.FunctionComponent<WebViewProps> = () => (
+  <View style={{ flex: 1 }}>
+    <Text>WebView</Text>
+  </View>
+);
```

Comment:
Can you explain what you're doing here?

Reply:
I wanted to have something rendered, this is actually not needed. I'm not actually sure if we need to have "WebView" component at all in `src/`, since the native implementations will anyway get used and we have anyway separate types under root.

I removed `src/WebView.tsx` completely now.

Reply:
Yeah, this file looks extraneous. Thanks!

Reply:
@jamonholmgren, actually the `src/WebView.tsx` is indeed needed to support Expo:

https://docs.expo.io/versions/latest/sdk/webview/

So what we basically need to do, is to create a WebView component which can be used by Expo - while the native ones that we have here are working for iOS Simulator & Android Studio emulator & etc just fine.

Reply:
![Screenshot 2020-05-07 at 9 52 02](https://user-images.githubusercontent.com/7287118/81263521-78fb4680-9048-11ea-99e8-49f9ea617a19.png)

Reply:
I tested with latest Expo, and it's exactly so.

Reply:
Actually with the previous version with that "dummy" `WebView.tsx`, Expo works in Android/iPhone versions and displays "WebView" in web version.

Reply:
Here's the Expo test setup: https://github.com/jussikinnula/react-native-webview-expo-test

Reply:
The latest Expo SDK + react-native-webview 8.1.1 show empty box on web version. So basically we are not breaking anything if we re-add the dummy and remove the `<Text>WebView</Text>` part.

See that example here: https://github.com/jussikinnula/react-native-webview-expo-test/tree/current-latest-react-native-webview

And both examples with Expo work with iOS/Android. I have still yet to test Android Studio simulator and add the documentation parts, etc.

@jamonholmgren, what do you think?

Reply:
There's actually some styles defined for the empty View (that `react-native` WebView produces):

![Screenshot 2020-05-08 at 9 38 53](https://user-images.githubusercontent.com/7287118/81378391-e02ffe00-910f-11ea-97e6-eb751fb81d23.png)

I can match those of course if we want.

Reply:
The current implementation matching new implementation is now pushed, and can be tested with `npm i github:jussikinnula/react-native-webview#build` or from https://github.com/jussikinnula/react-native-webview-expo-test/tree/build2 (Expo SDK test with it).

Reply:
@jamonholmgren, please review the latest additions and comment if this is not clear. I added test results, and so far the only things missing greatly:

1) What version number this should be, this is a breaking change (maybe 10.0.0).
2) I have to test with real devices. I only have iOS devices in my possession, so someone might pop in and help with Android?
3) Add documentation. There's actually not that much changes from that perspective, rather than dropping legacy WebView from react-native. I can document that.

Test coverage is a bit thin in my opinion, but I guess I could have separate contribution on that end.